bucket/sonar-scanner.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/",
+    "description": "Scan code for Sonarqube when there is no specific scanner for your build system.",
+    "license": "LGPL-3.0-only",
+    "version": "4.2.0.1873",
+    "architecture": {
+        "64bit": {
+            "url": "https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.2.0.1873.zip",
```

Comment:
This is not 64bit. 

WIndows 64bit url is this: https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.0.0.1744-windows.zip

anyarch version only depends on java and could be used in any environment.

Remove architecture.64bit property.

Reply:
Thanks for catching this @Ash258! Fixed.